.github/workflows/docker.yml

```
@@ -23,40 +23,36 @@ jobs:
         run: |
           docker-compose --version
 
-      - name: Build docker
-        run: docker-compose build
+      - name: Build base image, start containers, and wait for services
+        run: |
+          docker-compose up --build -d
+          # Hang tight until MySQL appears to be working
+          docker-compose exec -T web urlwait mysql://root:kuma@mysql:3306/developer_mozilla_org 30
```

I did some light research on the `-T`. It's just good practice, right? Something you'd only ever do in GitHub Actions. 

Anyway, I think if we're going to wait for `mysql` shouldn't we also wait for `redis`?
```suggestion
          docker-compose exec -T web urlwait mysql://root:kuma@mysql:3306/developer_mozilla_org 30
          docker-compose exec -T web urlwait redis://redis:6379/3 30
```

---

@peterbe Yes, the `-T` is only a good practice for those environments when you don't need to allocate a terminal.

---

Yes, good point, we should wait for `redis` as well.